core/runtime.rs

```diff
@@ -1732,7 +1732,8 @@ pub mod tests {
 
   #[test]
   fn test_heap_limits() {
-    let create_params = v8::Isolate::create_params().heap_limits(0, 20 * 1024);
+    let create_params =
+      v8::Isolate::create_params().heap_limits(0, 3 * 1024 * 1024);
```

Comment:
Note: required startup heap limits have increased.

Reply:
Increased a lot 😬

Reply:
V8 is really inefficient on first startup. You can see in the benchmarks that the snapshot (so baseline heap size of Deno) has not really increased.